<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1098</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LibreriApp</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(250, 250, 250);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,9">
      <property name="spacing">
       <number>6</number>
      </property>
      <item alignment="Qt::AlignBottom">
       <widget class="QFrame" name="frame">
        <property name="styleSheet">
         <string notr="true">background-color:rgb(255, 255, 255); border-radius: 24; </string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>15</number>
         </property>
         <item alignment="Qt::AlignLeft|Qt::AlignBottom">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Volkswagen-Serial</family>
             <pointsize>28</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>LibreriApp</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hLayout" stretch="7,3">
        <property name="leftMargin">
         <number>15</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>15</number>
        </property>
        <property name="bottomMargin">
         <number>15</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="7,1">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QTableWidget" name="tabla">
            <property name="styleSheet">
             <string notr="true">background-color:rgb(255, 255, 255);</string>
            </property>
            <column>
             <property name="text">
              <string>ID</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Título</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Autor</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Fecha</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Páginas</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Editorial</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Precio</string>
             </property>
            </column>
           </widget>
          </item>
          <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
           <widget class="QFrame" name="frame_4">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="agregarLibro">
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>50</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>25</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">border-radius: 24; background-color:rgb(35, 197, 94)</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>plus.png</normaloff>plus.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="9,0,1,1">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QListWidget" name="carrito">
            <property name="styleSheet">
             <string notr="true">background-color:rgb(255, 255, 255);</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="quitarSeleccion">
              <property name="styleSheet">
               <string notr="true">background-color:rgb(254, 120, 75);</string>
              </property>
              <property name="text">
               <string>Quitar seleccion</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="quitarTodos">
              <property name="styleSheet">
               <string notr="true">background-color:rgb(254, 120, 75)</string>
              </property>
              <property name="text">
               <string>Quitar todos</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item alignment="Qt::AlignRight">
           <widget class="QFrame" name="frame_3">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="total">
               <property name="font">
                <font>
                 <family>Volkswagen-Serial</family>
                 <pointsize>16</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Total</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_2">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="comprar">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>50</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Volkswagen-Serial</family>
                 <pointsize>12</pointsize>
                 <weight>50</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                 <underline>false</underline>
                 <strikeout>false</strikeout>
                 <kerning>true</kerning>
                </font>
               </property>
               <property name="focusPolicy">
                <enum>Qt::StrongFocus</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">border-radius:  24; background-color:rgb(35, 197, 94)</string>
               </property>
               <property name="text">
                <string>Comprar</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
